code/utility/logger.cpp

```diff
@@ -11,6 +11,7 @@ void Logger::setStream(QTextEdit* log_stream) {
 
 //TODO: Overload ostream operator
 bool Logger::log (std::string message, LogType type) {
+    if (type == LogType::DEBUG && !Logger::m_debug) return false;

```

Comment:
Hmm, do you know if there are any other flags that are defined for debug? It is preferable to avoid Qt specific stuff, in case we move over to cmake. Otherwise, we can leave this as is, just add a `TODO` comment to change the implementation later on.